### DISCUSSION TITLE
[FEATURE] <friendly ui for user and the color is too much bad and the button is look soo basic >

### DISCUSSION THREAD

**Discussion by sanaullah1266** [Dun-sin/Whisper#695] [Ideas]
### Description

the ui of the project is not that much good i want to add soo good animation and good buttons and user friendly features for the good user experience to the user

### Screenshots

![image](https://github.com/user-attachments/assets/becb62bb-ff4c-4015-a23b-66427e97f68d)

### Additional information

improve the ui is important for good user experience

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

**Comment by Dun-sin** [comment]
@sanaullah1266 can you provide the change you want to performa, your issue isn't descriptive enough, what do you want to add, Why and a visual showing the difference 

Example:
- the background color is dull, we should use white instead (then provide a screenshot of how white would look like in respect to the website)

**Reply by sanaullah1266** [reply]
Yeah for sure
